server/client_manager.py

```diff
@@ -1522,18 +1524,20 @@ def remove_client(self, client):
             if c.following == client:
                 c.unfollow()
         self.clients.remove(client)
+        server_count = 0
         for hub in self.server.hub_manager.hubs:
             count = 0
             for c in hub.clients:
                 if not c.area.hide_clients and not c.hidden:
                     count = count + 1
             hub.count = count
+            server_count = server_count + hub.count
         for c in self.server.client_manager.clients:
             if c.viewing_hub_list:
                 c.send_command(
                     "FA",
                     *[
-                        "{ Hubs }\n Double-Click me to see Areas\n  _______",
+                        f"{self.server.config['masterserver_name']}\n Users Online: {server_count}\n Double-Click me to see Areas\n _______",
```

Comment:
Not sure I like this. What if masterserver_name is not defined, or the server is not listed on the master server?